### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by CedrikNikita**
Should be well tested before merge

### CODE REVIEW

File: src/views/TipCommentsView.vue [hunk 1]
```diff
@@ -145,12 +145,9 @@ export default {
...
       if (this.USE_DEEP_LINKS) {
-        const url = new URL(`${process.env.VUE_APP_WALLET_URL}/comment`);
-        url.searchParams.set('id', this.tip.id);
-        url.searchParams.set('text', this.newComment);
-        url.searchParams.set('x-success', window.location);
-        url.searchParams.set('x-cancel', window.location);
-        window.location = url;
+        window.location = createDeepLinkUrl(
+          { type: 'comment', id: this.tip.id, text: this.newComment },
+        );
```
Comment (venimus) [inline src/views/TipCommentsView.vue:150]:
```suggestion
        window.location = createDeepLinkUrl(
          { type: 'comment', id: this.tip.id, text: this.newComment, }
        );
```
or better

```suggestion
        window.location = createDeepLinkUrl({
          type: 'comment',
          id: this.tip.id,
          text: this.newComment,
        });
```

File: src/components/TipInput.vue [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <template>
   <a
     v-if="USE_DEEP_LINKS && !userAddress"
-    :href="deepLink"
+    :href="createDeepLinkUrl({ type: isRetip ? 'retip' : 'tip', id: tip.id })"
```
Comment (venimus) [inline src/components/TipInput.vue:4]: I would have kept the deepLink function and put the logic there

Comment (venimus) [reply src/components/TipInput.vue:4]: Too soon... #439

**Comment by mradkov** [comment]
Can you sync with `develop` and resolve the conflicts? @CedrikNikita , thanks!

File: src/views/TipCommentsView.vue (hunk 1 above)
Comment (venimus) [reply src/views/TipCommentsView.vue:150]: Why? There is no second parameter IMHO this is less right than some formatting
I guess this is some bug in the es-linter when formatting this (weird) way

**Pull request merged**